Shared/Samples/Add dynamic entity layer/AddDynamicEntityLayerView.swift

```diff
@@ -75,4 +75,29 @@ struct AddDynamicEntityLayerView: View {
                 }
             }
     }
+    
+    @ViewBuilder private var settingsButton: some View {
+        Button("Dynamic Entity Settings") {
+            isShowingSettings = true
+        }
+    }
+    
+    @ViewBuilder private var settingsSheet: some View {
+        if #available(iOS 16, *),
+           horizontalSizeClass == .compact,
+           verticalSizeClass == .regular {
+            settingsButton
+                .sheet(isPresented: $isShowingSettings) {
+                    SettingsView()
+                        .environmentObject(model)
+                        .presentationDetents([.fraction(0.4)])
+                }
+        } else {
+            settingsButton
+                .sheet(isPresented: $isShowingSettings, detents: [.medium]) {
+                    SettingsView()
+                        .environmentObject(model)
+                }
+        }
+    }
```

Comment:
> But what's the harm in providing presentation regardless of the size class?

The sample will present better on iPad and Mac Catalyst with the size class check since the dynamic entities are visible when the [custom sheet](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/main/Shared/Supporting%20Files/Extensions/View%2BSheet.swift) is presented.

<img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/2b278718-84c3-4e92-b0c1-60eee4e431b9" width="500">
<img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/a7ac18c0-ba27-4a7e-8723-696981ecc132" width="500">